api/swagger.yaml

```diff
@@ -2260,7 +2260,9 @@ definitions:
                     ConfigName is the name of the config that this references, but this is just provided for
                     lookup/display purposes. The config in the reference will be identified by its ID.
                   type: "string"
-
+          Isolation:
+            description: "Isolation mode of the containers running the service (default, process, hyperv..."
```

Comment:
Perhaps the same terminology as is used on `docker run`, and add a note that it's only used for Windows containers;

```
description: |
  Container isolation technology to use for the container.

  <p><br /></p>

  > **Note**: Container isolation is only used for Windows containers.
  > This field is ignored for Linux containers.
```

(the `<p><br /></p>` is a bit hacky, but I haven't found another way to add spacing yet :smile:)